src/app/backend/resource/secret/secretdetail.go

```diff
@@ -36,9 +36,6 @@ type SecretDetail struct {
 
 	// Used to facilitate programmatic handling of secret data.
 	Type api.SecretType `json:"type"`
-
-	// Determines if the secret is hidden type in case of opaque and basicAuth types.
-	IsHiddenType bool `json:"isHiddenType"`
```

Comment:
kubectl describe does obfuscate namespace, so we do here. I'd say to obfuscate everything for now and when feature requests come to show individual types/fields, we'd show them. But hiding everything by default is safer.